lib/streamlit/elements/multiselect.py

```diff
@@ -112,6 +113,14 @@ def _check_and_convert_to_indices(options, default_values):
                 else:
                     default_values = list(default_values)
 
+            if not isinstance(options, list):
+                if is_type(options, "numpy.ndarray") or is_type(
+                    options, "pandas.core.series.Series"
+                ):
+                    options = list(options)
+                else:
+                    options = list(options)
```

Comment:
It seems like this could be just `options = list(options)`, since the branches are the same.